jenkins-integration/beaker/install/foss/40_clone_test_catalogs.rb

```diff
@@ -0,0 +1,16 @@
+test_name 'Clone test-catalogs repo onto master'
+
+# This is awful - we clone locally and then SCP the whole thing over.
+# See QENG-2556 for adding a proper way to do this in Beaker.
+tmp_repo = "/tmp/test-catalogs-#{Process.pid}"
+target = '/root/test-catalogs'
+
+teardown do
+  %x(rm -rf #{tmp_repo})
```

Comment:
`rm -rf ${var}`: that's not scary at all!  :)

Might be worth doing this through Ruby's File API just to make it a bit safer?  Or maybe at least guard this with `File.directory?(tmp_repo)`?